bazel/repositories.bzl

```diff
@@ -82,7 +93,16 @@ def proxy_wasm_cpp_host_repositories():
         patch_args = ["-p1"],
     )
 
-    # Core.
+    # Core deps. Try to keep up with Envoy versions:
+    # https://github.com/envoyproxy/envoy/blob/main/bazel/repository_locations.bzl
+
+    maybe(
+        http_archive,
+        name = "com_google_absl",
+        sha256 = "987ce98f02eefbaf930d6e38ab16aa05737234d7afbab2d5c4ea7adbe50c28ed",
+        strip_prefix = "abseil-cpp-20230802.1",
+        urls = ["https://github.com/abseil/abseil-cpp/archive/20230802.1.tar.gz"],
+    )
```

Comment:
Can this be a separate change or does it need to be in this PR? I doubt it affects generation of Bazel rules.

Reply:
I anticipated your comment :-) Will split this off once done (should be ready now, pending CI).

Reply:
https://github.com/proxy-wasm/proxy-wasm-cpp-host/pull/400

Reply:
Now split off entirely.